import-export-cli/cmd/aws/init.go

```diff
@@ -448,6 +447,8 @@ func initializeProject() error {
 	def.Context = string(os.PathSeparator) + flagApiNameToGet
 
 	oas3ByteValue := v2.CreateEpConfigForAwsAPIs(def, path)
+	def.AdvertiseInformation.Advertised = true
+	def.AdvertiseInformation.Vendor = "AWS"
```

Comment:
Shall we add this vendor property with a small explanation to [1] as well, since it will be the sample that we ask users to refer to from WSO2 documentation?

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/integration/testdata/sample-api.yaml#L160